File: backend/WhatsAppOrderNotification.Api/Data/Customer.cs
namespace WhatsAppOrderNotification.Api.Data;

public class Customer
{
    public int Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public string Document { get; set; } = string.Empty;
    public string Phone { get; set; } = string.Empty;
    public CustomerStatus Status { get; set; } = CustomerStatus.Pendiente;
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}

File: frontend/src/types/customer.ts
export type CustomerStatus = "Pendiente" | "Listo";

export interface Customer {
  id: number;
  name: string;
  document: string;
  phone: string;
  status: CustomerStatus;
  createdAt: string;
}

export interface CustomerFormValues {
  name: string;
  document: string;
  phone: string;
}
